discord.js

```diff
@@ -84,14 +84,21 @@ client.on('message', msg => {
 
 function parseMentionsInMessage(message) {
 
-	// const matches = mention.match(/^<@!?(\d+)>$/g)
-	// console.log(matches)
-	// matches.replaceAll(mention=>{
-	// 	replace
-	// })
-
-	// TODO: from DiscordMessageObject(text="hello <@00000000000000001>") it should return String("hello @Username")
-	return message
+  var mentions = [];
+  message.mentions.users.forEach(user => {
+    mentions.push({"userId": user.id, "username": user.username, "discriminator": user.discriminator});
+  });
+  var messageContent = message.content;
```

Comment:
Isn't using `let` instead of `var` is the standard right now?

Reply:
Yep, mb, webstorm made it var for some reason.